### PR TITLE
Handle config and node creation errors.

### PR DESCRIPTION
**Pull request by rjstires**
## Purpose
Added error handling which carries over from the NodeBalancerCreate flow.

- To test Config failure; Create a config with an HTTPS protocol, and put jibberish for the key and cert fields.
- To test Node failure; Create a node with an address a fake IP.
- It's gross looking until we redesign the notices.
- I fixed buttons on the Label panel.
- Fixed an errors with regard to error filters.

### CODE REVIEW

File: src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerSummary/NodeBalancerSummary.tsx [hunk 1]
```diff
@@ -1,13 +1,11 @@
 import * as React from 'react';
-import {
-  withStyles,
-  StyleRulesCallback,
-  Theme,
-  WithStyles,
-} from 'material-ui';
+import { withStyles, StyleRulesCallback, Theme, WithStyles } from 'material-ui';
```
Comment (patthiel) [inline src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerSummary/NodeBalancerSummary.tsx]: I've been refactoring my code to use this import format (per elsewhere in the codebase):
```
import {
  withStyles,
  StyleRulesCallback,
  Theme,
  WithStyles,
} from 'material-ui';
```
Do we need to configure the ts lint rules if imports are inconsistent?

Comment (rjstires) [reply src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerSummary/NodeBalancerSummary.tsx]: I'm of the opinion that It's unnecessary to dictate to a team of professionals how this is handled.

File: src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerDetail.tsx [hunk 2]
```diff
@@ -208,7 +207,10 @@ class NodeBalancerDetail extends React.Component<CombinedProps, State> {
             exact
             path={`${path}/summary`}
             render={() =>
-              <NodeBalancerSummary nodeBalancer={nodeBalancer} />
+              <NodeBalancerSummary
+                nodeBalancer={nodeBalancer}
+                errorResponses={this.props.location.state.errors}
+              />
```
Comment (martinmckenna) [inline src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerDetail.tsx]: what is going on with this `errorResponses` prop? It's crashing the NodeBalancerSummary page

Comment (rjstires) [reply src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerDetail.tsx]: This issue was fixed in another branch, if you could review again please.

File: src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerSummary/NodeBalancerSummary.tsx (hunk 1 above)
Comment (asauber) [reply src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerSummary/NodeBalancerSummary.tsx]: I think if the import doesn't exceed our line length, it doesn't matter if the imports are on separate lines or not.

File: src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerDetail.tsx [hunk 3]
```diff
@@ -209,7 +208,10 @@ class NodeBalancerDetail extends React.Component<CombinedProps, State> {
             exact
             path={`${path}/summary`}
             render={() =>
-              <NodeBalancerSummary nodeBalancer={nodeBalancer} />
+              <NodeBalancerSummary
+                nodeBalancer={nodeBalancer}
+                errorResponses={this.props.location.state.errors}
```
Comment (asauber) [inline src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerDetail.tsx]: Upon viewing a NodeBalancer without any errors to display.

`Cannot read property 'errors' of undefined`

**Pull request merged**